recipes/semsimian/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "semsimian" %}
+{% set version = "0.2.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/monarch-initiative/semsimian/archive/v{{ version }}.tar.gz
+  sha256: 0ce33ab71a2795088a759fa88e932267c363dc8555c6ef14d6971f194b022f4d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - {{ stdlib('c') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - maturin >=0.14,<0.15
+    - pip
+  run:
+    - python
+    - libgcc-ng  # [linux]
+    - libcxx  # [osx]
+    - vs2015_runtime  # [win]
```

Comment:
These should be automatically added by the run_exports of the compilers.
```suggestion
```

Reply:
it seems the pipeline breaks w/o `vs2015_runtime  # [win]` while the other two went through fine without

Reply:
This shouldn't be the case. What was the error?